pom.xml

```diff
@@ -107,6 +108,12 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+   <groupId>ch.qos.logback</groupId>
```

Comment:
That's right logback dependency is not needed to compile as it uses slf4j.
I'm removing that.